addons/l10n_pk_edi/models/res_config_settings.py

```diff
@@ -0,0 +1,39 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, _
+from odoo.exceptions import RedirectWarning
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    l10n_pk_edi_pos_id = fields.Char("PoS ID", related="company_id.l10n_pk_edi_pos_id", readonly=False)
+    l10n_pk_edi_token = fields.Char("Pakistan EDI Token", related="company_id.l10n_pk_edi_token", readonly=False)
+    # This proxy is only to access FBR APIs from out side of Pakistan for testing purpose.
+    # And will remove before stable release. (Use proxy server of USA and Dubai to access)
+    l10n_pk_edi_proxy = fields.Char("HTTPS Proxy", related="company_id.l10n_pk_edi_proxy", readonly=False)
+
+    def l10n_pk_check_ntn_cnic(self):
+        if not self.l10n_pk_edi_pos_id:
+            action = {
+                "view_mode": "form",
+                "type": "ir.actions.act_window",
+                "res_id": self.company_id.id,
+                "res_model": "res.config.settings",
+                "views": [[self.env.ref("base.res_config_settings_view_form").id, "form"]]
+            }
+            raise RedirectWarning(_("Please enter a PoS ID in invoice section of configuration."),
+                action, _('Go to Configuration')
```

Comment:
```suggestion
            raise RedirectWarning(
                _("Please configure PoS ID under Pakistan Localisation"),
                action.id,
                 _('Go to Configuration')
```
Or may be better message just a suggestion